src/main/amp/config/alfresco/module/support-tools/module-context.xml

```diff
@@ -14,11 +14,11 @@
 	See the License for the specific language governing permissions and
 	limitations under the License.
 	
--->
-<!DOCTYPE beans PUBLIC '-//SPRING//DTD BEAN//EN' 'http://www.springframework.org/dtd/spring-beans.dtd'>
-
-<beans>
-	<!-- This is filtered by Maven at build time, so that module name is single sourced. -->
-    <import resource="classpath:alfresco/module/AdminConsoleSupportTools/context/service-context.xml" />
-    
-</beans>
+-->
+<!DOCTYPE beans PUBLIC '-//SPRING//DTD BEAN//EN' 'http://www.springframework.org/dtd/spring-beans.dtd'>
+
+<beans>
+	<!-- This is filtered by Maven at build time, so that module name is single sourced. -->
+    <import resource="classpath:alfresco/module/support-tools/context/service-context.xml" />
```

Comment:
I don't think this is going to work as we have not changed the folder name i.e. the folder src/main/amp/config/alfresco/module/AdminConsoleSupportTools.

However, as mentioned earlier REPO-494 was written in light of taking ownership of the whole project, for this release I think it's fine to leave this, I was more concerned about the public facing wording and you have that covered in this commit already so I would revert this change.